web_external/ImagesGallery/Detail/imageDetailsPage.jade

```diff
@@ -1,14 +1,33 @@
 .isic-image-details-container
-  .button
-    img(src='#{imgRoot}/ex.svg')
 
-  if image.id
-    .isic-image-details-name= image.name()
-      button.btn.btn-xs.btn-default.padleft.padbottom.zoom(data-toggle="tooltip" data-placement="bottom" title="View this image in full resolution")
+  .isic-image-details-heading
+    if image.id
+      span.isic-image-details-name= image.name()
+      button#isic-image-details-zoom.isic-image-details-button.btn.btn-xs.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='View this image in full resolution'
+      )
         i.icon-zoom-in
-      button.btn.btn-xs.btn-default.padleft.padbottom.openwindow(data-toggle="tooltip" data-placement="bottom" title="Open this image in a new window")
+      a#isic-image-details-openwindow.isic-image-details-button.btn.btn-xs.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='Open this image in a new window'
+        href='/api/v1/image/' + image.id + '/download?contentDisposition=inline'
+        target='_blank'
+      )
         i.icon-link-ext
+      a#isic-image-details-download.isic-image-details-button.btn.btn-xs.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='Download this image'
+        href='/api/v1/image/' + image.id + '/download'
```

Comment:
That's a good point. I actually thought about adding the attribute, but I didn't like having to redundantly add the file name there too. However, the docs also state
> If the HTTP header Content-Disposition: gives a different filename than this attribute, the HTTP header takes priority over this attribute.
so I can probably just add the attribute with no value. I'll try it now.